src/main/java/org/jenkinsci/plugins/docker/workflow/FromFingerprintStep.java

```diff
@@ -49,11 +54,13 @@
 
     private final String dockerfile;
     private final String image;
+    private final List<String> buildArgs;
```

Comment:
This would better be mutable and set in a `@DataBoundSetter`. Also it must have a getter. For completeness `config.jelly` should at least mention it, though pending [JENKINS-27901](https://issues.jenkins-ci.org/browse/JENKINS-27901) it cannot really be set there.